### DISCUSSION TITLE
New Destination: Singlestore

### DISCUSSION THREAD

**Discussion by bhaskar-pv** [airbytehq/airbyte#36202] [New Connector Request]
Hi Guys,

Any plan for the singlestore destination? I tried to use MySQL connector to use it for Singlestore but it didnt work. 

Any plan on singlestore destination? We are planning for "Power by Airbyte" if this will be available

**Comment by oyeliseiev-ua** [comment]
Hi,
Yes, I plan to develop it after source connector in near future. 
As a workaround you can use not published python connector: https://github.com/mohamagdy/airbyte/pull/2/files.

**Comment by marcosmarxm** [comment]
@oyeliseiev-ua developed the source https://github.com/airbytehq/airbyte/pull/37337

**Reply by oyeliseiev-ua** [reply]
destination implementation: [38600](https://github.com/airbytehq/airbyte/pull/38600/)

[a comment by an automated account is not shown]
